package.json

```diff
@@ -31,6 +31,8 @@
     "history": "2.1.1",
     "iflow-lodash": "^1.1.16",
     "iflow-moment": "^1.1.13",
+    "iflow-react-router": "^1.1.16",
```

Comment:
yeah, I'm not exactly sure why it didn't, but that is strange...maybe because these packages don't have any dependencies?